proto/api/v2/report_configuration_service.proto

```diff
@@ -1,14 +1,15 @@
 syntax = "proto3";
 
 option go_package = "v2";
-
 option java_package = "io.stackrox.proto.api.v2";
 
 import weak "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
 import "api/v1/common.proto";
 import "api/v1/empty.proto";
 import "api/v1/search_service.proto";
-import "google/protobuf/timestamp.proto";
+
+package v2;
```

Comment:
😱

Reply:
Looks like that is not required because the code generator still put it in the v2 package, but added it for consistency with other service protos.